src/components/NewBeer.js

```diff
@@ -0,0 +1,70 @@
+import './NewBeer.css';
+import { useState } from 'react';
+import axios from 'axios';
+import { Form } from 'react-router-dom';
+
+const NewBeer = ({random}) => {
+
+    const baseUrl = "https://ih-beers-api2.herokuapp.com/beers";
+    const [inputs, setInputs] = useState({});
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
```

Comment:
That one would prevent the user to send the form without the name, so it won't fail the request... And an error message would be good so the user understand what's happening

```suggestion
        e.preventDefault();
        if (!inputs.name) {
          // here you could setErrorMessage too so it would be more User Friendly
          return
        }
```